src/models/wasteWater/story/WasteWaterSamplingSitesHooks.ts

```diff
@@ -0,0 +1,74 @@
+import { WasteWaterDataset, WasteWaterTimeEntry } from '../types';
+import { useEffect, useMemo, useState } from 'react';
+import { getData } from '../loading';
+import { Utils } from '../../../services/Utils';
+import { sortBy } from '../../../helpers/lodash_alternatives';
+import { DateRange, isInDateRange } from '../../../data/DateRange';
+import dayjs from 'dayjs';
+
+export interface WasteWaterDataWithLocation {
+  variantsTimeseriesSummaries: {
+    data: WasteWaterTimeEntry[];
+    name: string;
+  }[];
+  location: string;
+}
+
+export function useWasteWaterData(): WasteWaterDataWithLocation[] | undefined {
+  const [wasteWaterData, setWasteWaterData] = useState<WasteWaterDataset | undefined>(undefined);
+  useEffect(() => {
+    getData({
+      country: 'Switzerland',
+    }).then(dataset => dataset && setWasteWaterData(dataset));
+  }, []);
+
+  const locationData = useMemo(() => {
```

Comment:
IntelliJ suggests to inline this:
```suggestion
  return useMemo(() => {
```